package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "zelagac",
+  "version": "1.0.0",
+  "description": "zelagac: ZEroLAbs GAs Consumption",
+  "main": "main.js",
+  "scripts": {
+    "test": "mocha --reporter spec"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mancap314/zelagac.git"
```

Comment:
Can we update these URLs to this repo? Thanks

Reply:
Hi, fixed in commit acb3fc079a39bd3663cb89fb90045d5c07a71607